### PR TITLE
Feature/87686 non attendance email tweaks

### PR DESCRIPTION
**Pull request by niconerd**
https://central.tri.be/issues/87686

Port changes [previously made in the tickets email](https://central.tri.be/issues/73885) to the non-attendance email.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | You must have a space after a closing parenthesis | [email-non-attendance.php:256](https://github.com/moderntribe/event-tickets/blob/bac035718dd59196a23489243f9599fe21b32cb4/src/views/tickets/email-non-attendance.php#L256)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$event_date' | [email-non-attendance.php:258](https://github.com/moderntribe/event-tickets/blob/bac035718dd59196a23489243f9599fe21b32cb4/src/views/tickets/email-non-attendance.php#L258)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$event_date' | [email-non-attendance.php:258](https://github.com/moderntribe/event-tickets/blob/243df161f59cc32d1332e0d62346a953ee1d5873/src/views/tickets/email-non-attendance.php#L258)

> via jenkins/codesniffer-phpcs

File: src/views/tickets/email-non-attendance.php [hunk 1]
```diff
@@ -12,10 +12,25 @@
...
+ * @version TBD
  */
 
-$start_date = function_exists( 'tribe_get_start_date' ) ? tribe_get_start_date( $event_id ) : null;
+$event_date = null;
+
+/**
+ * Filters whether or not the event date should be included in the ticket email.
+ *
+ * @since 4.5.11
```
Comment (bordoni) [inline src/views/tickets/email-non-attendance.php]: TBD

File: src/views/tickets/email-non-attendance.php [hunk 2]
```diff
@@ -12,10 +12,25 @@
...
+ *
+ * @var bool Include event date? Defaults to false.
```
Comment (bordoni) [inline src/views/tickets/email-non-attendance.php]: use `@param` not `@var`

**Pull request merged**